client/src/about/components/About.tsx

```diff
@@ -1,13 +1,19 @@
-import React, { FC } from "react";
+import React, { FC } from 'react';
+import { sendUpdateMessage, sendLoginMessage } from '../../websocket/actions'
 import "./About.scss";
-import { sendUpdateMessage } from "../../websocket/actions";
 
 interface Props {
   sendUpdateMessage: (id, x, y, r, g, b) => void;
+  sendLoginMessage: (id) => void;
 }
 
-const About: FC<Props> = ({ sendUpdateMessage }) => (
+const About: FC<Props> = ({ sendUpdateMessage, sendLoginMessage }) => (
   <div className='about-page'>
+    <h2>TESTING AREA</h2>
+      <p>Click "Pixel 1" to send an update message to the server!
+      </p>
+        <button onClick = {() => sendUpdateMessage("user1", 10, 400, 255, 255, 255)} id="p1"> Pixel 1 </button>
+        <button onClick = {() => sendLoginMessage("testemail@gmail.com")} id="login"> User Login </button>
```

Comment:
I feel like this could be put on a separate "testing" page, but I can move it for y'all after you've merged this if you want.

Reply:
this is our testing page, afaik no one else is using this page? we can rename it to Testing though

Reply:
I mean if you want to make a separate testing page that would be great @almagee16 . We were just piggy-backing off of the about page and were going to remove this later